woocommerce/woocommerce-ios
# Update fastlane

This PR updates `Fastlane` to get rid of the `mini_magic` security alert.

To test:

1. Run `bundle install`.
2. Run one of the release lanes  and verify that it works.

Update release notes:
